code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1210,13 +1210,37 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	switch(H.nutrition)
 		if(NUTRITION_LEVEL_FULL to INFINITY)
 			H.throw_alert("nutrition", /obj/screen/alert/fat)
+			var/obj/item/organ/O = H.getorganslot(ORGAN_SLOT_HEART)
+			O.applyOrganDamage(0.25)
 		if(NUTRITION_LEVEL_HUNGRY to NUTRITION_LEVEL_FULL)
 			H.clear_alert("nutrition")
+			for(var/slot in list(ORGAN_SLOT_BRAIN, ORGAN_SLOT_HEART, ORGAN_SLOT_LUNGS, ORGAN_SLOT_APPENDIX, \
+	ORGAN_SLOT_EYES, ORGAN_SLOT_EARS, ORGAN_SLOT_TONGUE, ORGAN_SLOT_LIVER))
```

Comment:
damn, I missed my lunch break, now my internal organs are going to literally rot inside of me, and when I die from said organ rotting, my corpse will be about as hard to resurrect as a corpse that has been rotting for 15+ minutes

like, can we have some admins do some checks using varedit (or whatever allows them to see the variables of certain creatures) on random people during a round to see if your claims of people ignoring the hunger system are actually true? because if they're not, why are we doing something drastic that will affect the balance/functionality of several items and mechanics?

Reply:
get over it?